tiledb/sm/cpp_api/array_schema.h

```diff
@@ -157,6 +157,18 @@ class ArraySchema : public Schema {
   /*                API                */
   /* ********************************* */
 
+  /**
```

Comment:
#ifndef TILEDB_REMOVE_DEPRECATIONS
```suggestion
  /**
```